.github/workflows/sandbox.yml

```diff
@@ -2,31 +2,46 @@ name: Build & Push Sandbox Docker Image
 
 on:
   pull_request:
-  create:
-    tags:
-      - v*
+  release:
+    types: [published]
 
 jobs:
-  push-github:
-    name: Push to GHCR
+  push-sandbox-image:
+    name: Push sandbox image to GHCR
     runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: cookbook
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: "0"
-      - name: Push Sandbox Plugin Docker Image to Github Registry
+      - name: Push Sandbox Docker Image to Github Registry
         uses: whoan/docker-build-with-cache-action@v5
         with:
           # https://docs.github.com/en/packages/learn-github-packages/publishing-a-package
           username: "${{ secrets.FLYTE_BOT_USERNAME }}"
           password: "${{ secrets.FLYTE_BOT_PAT }}"
           image_name: ${{ github.repository_owner }}/flyte-sandbox
           image_tag: latest,${{ github.sha }},${{ github.event.ref }}
-          push_git_tag: true
```

Comment:
Are you removing this just for testing?

I think ideally we would replicate this workflow, one that get triggered with pull_request and does no push_git_tag and another with release creation that does have `push_git_tag: true`...

Reply:
ah I just noticed the condition you added! awesome

Reply:
Well, no. I think this is unnecessary with `image_tag: ${{ github.sha }}`, right? Am I missing something here?